src/App/Pages/Accounts/LoginSsoPageViewModel.cs

```diff
@@ -268,18 +276,14 @@ private async Task LogIn(string code, string codeVerifier, string orgId)
                 // In the standard, non TDE case, a user must set password if they don't
                 // have one and they aren't using key connector.
                 // Note: TDE & Key connector are mutually exclusive org config options.
-                if (response.ResetMasterPassword || (decryptOptions?.RequireSetPassword ?? false))
+                if (response.ResetMasterPassword || (decryptOptions?.RequireSetPassword == true))
                 {
+                    // TODO: We need to look into how to handle this when Org removes TDE
+                    // Will we have the User Key by now to set a new password?
```

Comment:
❓ Is there a task for this or should it be part of this PR?

Reply:
@jlf0dev ?

Reply:
We have a followup task to look into TDE unenroll and how to migrate users. Right now it doesn't work but wanted to add a note